src/machines/should-i-train-machine.tsx

```diff
@@ -0,0 +1,161 @@
+import {assign, createMachine} from 'xstate'
+import {
+  Question,
+  getNextQuestion,
+  getPrevQuestion,
+} from 'utils/generate-question-states'
+import quiz from 'data/should-i-train.json'
+import {find} from 'lodash'
+
+export interface ShouldITrainQuizMachineContext {
+  responses: Score[]
+  currentQuestion: Question
+}
+
+interface Score {
+  score: number
+  selectedResponse: string
+  question: string
+  questionId: string
+}
+
+export type ShouldITrainQuizMachineEvent =
+  | {
+      type: 'BACK'
+    }
+  | {
+      type: 'CONFIRM'
+      data: Score
+    }
```

Comment:
only two actions, back and confirm. We dont actually use back right now...

Reply:
I assume this would be very similar in implementing to `assignCurrentQuestion`